packages/stream-server-uws/README.md

```diff
@@ -0,0 +1,41 @@
+# Meteor's DDP stream server based on uWS
+
+## Installation
+
+1. Install the fork of the uWebSocket Node.js bindings:
+
+    ```bash
+    meteor npm install sc-uws
```

Comment:
How is [`sc-uws`](https://www.npmjs.com/package/sc-uws) different from the deprecated [`uws` npm package](https://www.npmjs.com/package/uws), other than not being deprecated? Relatedly, where is the repository from which `sc-uws` gets `npm publish`ed (the GitHub links still point to the old project)? I want to make sure this `sc-uws` package is actively maintained, and not just a fork of the deprecated code.

Reply:
`uws` package "deprecated" because the author don't like `semver` policy of `npm`.

> How is sc-uws different from the deprecated uws npm package, other than not being deprecated?

It have the last stable uWebSockets version, without strange changes, like deleting of `http.h` or `addon.h` in bindings.

> Relatedly, where is the repository from which sc-uws gets npm published (the GitHub links still point to the old project)?

https://github.com/SocketCluster/uWebSockets

>  I want to make sure this `sc-uws` package is actively maintained, and not just a fork of the deprecated code.

There is no actively maintained package of uWebSockets.

Maybe Alex Hulman @alexhultman publish new package with Node.js binding, maybe not.